frontend/src/components/SearchPage/CourseItem.js

```diff
@@ -130,6 +130,8 @@ const CourseMetric = styled.div`
   text-align: center;
 `;
 
+const blocked = (style, size = '5px') => styled(style)`filter: blur(${size});`;
```

Comment:
Nitpick -- the 'style' argument should probably be 'component' (the signature of the `styled` function is `Component -> Component`)